src/generated/resources/assets/woodworks/lang/ja_jp.json

```diff
@@ -0,0 +1,58 @@
+{
```

Comment:
This file should be in `src/main/resources/` not `src/generated/resources` because it was not generated using datagen.